src/Spectre.libGaClassifier/ClassifierFactory.h

```diff
@@ -0,0 +1,58 @@
+/*
+* ClassifierFactory.h
+* The factory for creating classifiers
+*
+Copyright 2018 Spectre Team
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+#include "Spectre.libClassifier/Svm.h"
+#include "Spectre.libGenetic/DataTypes.h"
+#include "GaClassifier.h"
+
+namespace spectre::supervised
+{
+/// <summary>
+/// The factory for creating classifiers
+/// </summary>
+class ClassifierFactory
```

Comment:
Static class maybe? This is yet another class that does not store any state, so I think this will be a suitable approach.